### PR TITLE
Add key prop to video source

### PR DESCRIPTION
**Pull request by MatixYo**
On source elements inside the videos, the key prop is missing causing errors.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -548,7 +548,7 @@ class Cropper extends React.Component<CropperProps, State> {
               controls={false}
             >
               {(Array.isArray(video) ? video : [{ src: video }]).map((item) => (
-                <source {...item} />
+                <source key={item.type} {...item} />
```
Comment (ValentinH) [inline src/index.tsx:551]: Type is missing for the default case where video is not an array.

Comment (MatixYo) [reply src/index.tsx]: Switched to item.src as a key which should be unique.

**Review by ValentinH — APPROVED** [review]

**Comment by ValentinH** [comment]
Thanks published as 3.5.1: https://github.com/ricardo-ch/react-easy-crop/releases/tag/v3.5.1

**Comment by ValentinH** [comment]
@all-contributors please add @MatixYo  for code

**Pull request merged**